src/api/usersRouter.ts

```diff
@@ -19,25 +19,8 @@ import {
 
 export function createUsersRouter() {
   const usersRouter = express.Router();
-  const emailParser = z.object({
-    email: z.string().optional(),
-  });
-
-  usersRouter.get('/', async (req, res) => {
```

Comment:
I've changed this to /current - nothing else was finding by email and nothing should be able to load the full list at this stage.